pages/news/2020-07-16-portable_mixxx_setup.html

```diff
@@ -0,0 +1,74 @@
+
+### Putting one and two together
+
+Being both a controller and audio interface, The Akiyama Pulsar reduced the overhead created by multiple USB devices (?) allowing the Raspberry Pi to run Mixxx smoothly with < 20ms latency and no underruns. Note, this is without any optimalisations to the Rasbian OS (realtime kernel? jack?) so there is possibly room for improvement!
+
+### Mapping
+
+As there was no mapping for the controller I set out to create my own. Most buttons are easily mapped usng the GUI, but the jogwheels can be problematic. Departing from an existing community contributed mapping script that did not intimidate me (Icon iDJ) and by launching Mixxx with the '--controllerDebug' argument I was able to atleast get the jogwheels to be somewhat reactive...
+
+### Skinning the fat
+
+To make the setup extra portable I dressed my Raspberry Pi in a 2.8" Touch Screen that is long out of production. Being rather impractical, I would suggest getting a larger screen. Nevertheless, you will need a skin that accomodates the data you are interested in, while being readable on a small screen. Again I started of from an existing skin, an older 'minimal' skin found on in an old Mixxx install (< 2.0.0) only to find out there is no scaling, only absolute values for size and position as well a different naming convention for buttons, making it incompatible with newer skins and mappings ('library' verus 'playlist' for example). The skins provided by the later Mixxx distributions seemed rather indimidating at first but soon made me realized they are far more flexible than they look. Next to dynamic scaling using '(e,e)' units the newer default skins are somewhat modular. Departing from LateNight i simply deleted the style xml files for the buttons/sections I did not need, ending up with a minimal two deck view of waveforms, track name, bpm and using a leftover key on my controller to show and hide the fullscreen library! 
+
+### Confusion
+
+The paths where Mixxx stores it's default skins and controller mappings are different on various Operating Systems. Working with both OSX and Rasbian this was somewhat confusing at times. To add to this, the folder of DEFAUT skins/mappings is not the same as the 'user' skins/mappings. As soon as you edit a default mapping for example, it gets copied to your 'user' settings folder (sometimes under another name?). Keep this in mind (and just edit/copy your custom mapping directly to your 'user' settings folder).
+
+### In short
+
+* Get a Raspberry Pi 3 Model B or newer
+* Find a USB DJ controller with built-in sound-card.
+* Get a powerbank that can deliver the right amount of current. The specifications advertised/printed most electronics don't seem to tell much, there is only one way to find out...try it out and let the [community](https://mixxx.zulipchat.com/) know!
+
+
+### Read more
```

Comment:
I'd suggest to swap this with the demo video and put the read more paragraph at the end.